com.spotify.EdgeClient.json

```diff
@@ -0,0 +1,276 @@
+{
+    "app-id": "com.spotify.EdgeClient",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
```

Comment:
```suggestion
    "runtime-version": "23.08",
```

Reply:
this question still applies. Using obsolete runtime always lead to have them stay that way for longer than necessary.